Applications/CMC/test/testCMCGait10dof18musc.cpp

```diff
@@ -66,63 +66,9 @@ void testGait10dof18musc() {
-    rms_tols[5]  = 0.0001;   // ground_pelvis/pelvis_ty/speed
-    rms_tols[6]  = 0.00006;  // hip_r/hip_flexion_r/value
-    rms_tols[7]  = 0.002;    // hip_r/hip_flexion_r/speed
-    rms_tols[8]  = 0.00006;  // hip_l/hip_flexion_l/value
-    rms_tols[9]  = 0.002;    // hip_l/hip_flexion_l/speed
-    rms_tols[10] = 0.00004;  // back/lumbar_extension/value
-    rms_tols[11] = 0.001;    // back/lumbar_extension/speed
-    rms_tols[12] = 0.00005;  // knee_r/knee_angle_r/value
-    rms_tols[13] = 0.001;    // knee_r/knee_angle_r/speed
-    rms_tols[14] = 0.00008;  // knee_l/knee_angle_l/value
-    rms_tols[15] = 0.0007;   // knee_l/knee_angle_l/speed
-    rms_tols[16] = 0.00003;  // ankle_r/ankle_angle_r/value
-    rms_tols[17] = 0.003;    // ankle_r/ankle_angle_r/speed
-    rms_tols[18] = 0.0001;   // ankle_l/ankle_angle_l/value
-    rms_tols[19] = 0.003;    // ankle_l/ankle_angle_l/speed
-    rms_tols[20] = 0.008;    // hamstrings_r/activation
-    rms_tols[21] = 0.00005;  // hamstrings_r/fiber_length
-    rms_tols[22] = 0.0007;   // bifemsh_r/activation
-    rms_tols[23] = 0.00001;  // bifemsh_r/fiber_length
-    rms_tols[24] = 0.004;    // glut_max_r/activation
-    rms_tols[25] = 0.00002;  // glut_max_r/fiber_length
-    rms_tols[26] = 0.0007;   // iliopsoas_r/activation
-    rms_tols[27] = 0.00001;  // iliopsoas_r/fiber_length
-    rms_tols[28] = 0.00001;  // rect_fem_r/activation
-    rms_tols[29] = 0.000001; // rect_fem_r/fiber_length
-    rms_tols[30] = 0.002;    // vasti_r/activation
-    rms_tols[31] = 0.00001;  // vasti_r/fiber_length
-    rms_tols[32] = 0.005;    // gastroc_r/activation
-    rms_tols[33] = 0.00001;  // gastroc_r/fiber_length
-    rms_tols[34] = 0.0009;   // soleus_r/activation
-    rms_tols[35] = 0.00001;  // soleus_r/fiber_length
-    rms_tols[36] = 0.0002;   // tib_ant_r/activation
-    rms_tols[37] = 0.00001;  // tib_ant_r/fiber_length
-    rms_tols[38] = 0.0008;   // hamstrings_l/activation
-    rms_tols[39] = 0.00001;  // hamstrings_l/fiber_length
-    rms_tols[40] = 0.002;    // bifemsh_l/activation
-    rms_tols[41] = 0.00001;  // bifemsh_l/fiber_length
-    rms_tols[42] = 0.0002;   // glut_max_l/activation
-    rms_tols[43] = 0.00001;  // glut_max_l/fiber_length
-    rms_tols[44] = 0.0007;   // iliopsoas_l/activation
-    rms_tols[45] = 0.00001;  // iliopsoas_l/fiber_length
-    rms_tols[46] = 0.0004;   // rect_fem_l/activation
-    rms_tols[47] = 0.00001;  // rect_fem_l/fiber_length
-    rms_tols[48] = 0.0004;   // vasti_l/activation
-    rms_tols[49] = 0.00001;  // vasti_l/fiber_length
-    rms_tols[50] = 0.002;    // gastroc_l/activation
-    rms_tols[51] = 0.00001;  // gastroc_l/fiber_length
-    rms_tols[52] = 0.002;    // soleus_l/activation
-    rms_tols[53] = 0.00003;  // soleus_l/fiber_length
-    rms_tols[54] = 0.004;    // tib_ant_l/activation
-    rms_tols[55] = 0.00004;  // tib_ant_l/fiber_length
+    Array<double> rms_tols(0.035, 2*10+2*18); // Activations within 2%.
```

Comment:
It may be the case that `Storage::compareColumnRMS()` should be numerically integrating the RMS. That is, something like`trapz(time, diff * diff) / (endTime - startTime)` I see that it is *somewhat* doing this "integration" by dividing by the `endIndex - startIndex`, but this does not account for non-uniform time steps.